Tests/configure_and_test_integration_instances.py

```diff
@@ -758,16 +759,20 @@ def main():
 
     installed_content_packs_successfully = True
 
-    if server_version_compare(server_numeric_version, '6.0') >= 0:
+    if LooseVersion(server_numeric_version) >= LooseVersion('6.0.0'):
         with open('./Tests/content_packs_to_install.txt', 'r') as packs_stream:
             pack_ids = [pack_id.rstrip('\n') for pack_id in packs_stream.readlines()]
-        client = demisto_client.configure(base_url=servers[0], username=username, password=password,
-                                          verify_ssl=False)
-        installed_content_packs_successfully, installed_packs = search_and_install_packs_and_their_dependencies(
-            pack_ids, client, prints_manager
-        )
-        installed_packs_message = "Installed the content packs: {}".format(installed_packs)
-        prints_manager.add_print_job(installed_packs_message, print_color, 0, LOG_COLORS.GREEN)
+
+        # install content packs in every server
+        try:
+            for server_url in servers:
+                client = demisto_client.configure(base_url=server_url, username=username, password=password,
+                                                  verify_ssl=False)
+                search_and_install_packs_and_their_dependencies(pack_ids, client, prints_manager)
+        except Exception as e:
+            prints_manager.add_print_job(str(e), print_error, 0)
+            prints_manager.execute_thread_prints(0)
+            installed_content_packs_successfully = False
```

Comment:
let's check this together.
I think "search_and_install_packs_and_their_dependencies" should return if it has installed secsesfully. or not.

also, we need to print the list of packs that search_and_install_packs_and_their_dependencies, exempted to install.(including the ones the server suggested)